config.go

```diff
@@ -10,112 +10,105 @@ const (
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) SetIntConfigValueForKey(key string, value int) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetIntConfigValueForKey(key string, value int) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) SetLongConfigValueForKey(key string, value int64) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetInt32ConfigValueForKey(key string, value int32) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) SetStringConfigValueForKey(key, value string) {
-	err := lw.SaveUserConfigValue(key, value)
-	if err != nil {
-		log.Errorf("error setting config value: %v", err)
-	}
+func (mw *MultiWallet) SetLongConfigValueForKey(key string, value int64) {
+	mw.SaveUserConfigValue(key, value)
 }
 
-func (lw *LibWallet) ReadBoolConfigValueForKey(key string) (valueOut bool) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
-	if err != nil {
-		log.Errorf("error reading config value: %v", err)
-	}
+func (mw *MultiWallet) SetStringConfigValueForKey(key, value string) {
+	mw.SaveUserConfigValue(key, value)
+}
+
+func (mw *MultiWallet) ReadBoolConfigValueForKey(key string, defaultValue bool) (valueOut bool) {
+	mw.ReadUserConfigValue(key, &valueOut, defaultValue)
 	return
 }
 
-func (lw *LibWallet) ReadDoubleConfigValueForKey(key string) (valueOut float64) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
-	if err != nil {
-		log.Errorf("error reading config value: %v", err)
-	}
+func (mw *MultiWallet) ReadDoubleConfigValueForKey(key string, defaultValue float64) (valueOut float64) {
+	mw.ReadUserConfigValue(key, &valueOut, defaultValue)
 	return
 }
 
-func (lw *LibWallet) ReadIntConfigValueForKey(key string) (valueOut int) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
-	if err != nil {
-		log.Errorf("error reading config value: %v", err)
-	}
+func (mw *MultiWallet) ReadIntConfigValueForKey(key string, defaultValue int) (valueOut int) {
+	mw.ReadUserConfigValue(key, &valueOut, defaultValue)
 	return
 }
 
-func (lw *LibWallet) ReadLongConfigValueForKey(key string) (valueOut int64) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
-	if err != nil {
-		log.Errorf("error reading config value: %v", err)
-	}
+func (mw *MultiWallet) ReadInt32ConfigValueForKey(key string, defaultValue int32) (valueOut int32) {
+	mw.ReadUserConfigValue(key, &valueOut, defaultValue)
 	return
 }
 
-func (lw *LibWallet) ReadStringConfigValueForKey(key string) (valueOut string) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
-	if err != nil {
-		log.Errorf("error reading config value: %v", err)
-	}
+func (mw *MultiWallet) ReadLongConfigValueForKey(key string, defaultValue int64) (valueOut int64) {
+	mw.ReadUserConfigValue(key, &valueOut, defaultValue)
```

Comment:
As suggested in https://github.com/raedahgroup/dcrlibwallet/pull/57/files#r343517646, this should be
```
return (mw.ReadUserConfigValue(key, &valueOut, defaultValue)).(int64)
```